github/gists.go

```diff
@@ -211,6 +234,29 @@ func (s *GistsService) Edit(id string, gist *Gist) (*Gist, *Response, error) {
 	return g, resp, err
 }
 
+// List gist commits.
+//
+// Github API docs: https://developer.github.com/v3/gists/#list-gist-commits
+
+func (s *GistsService) ListCommits(id string) ([]*GistCommit, *Response, error) {
+	u := fmt.Sprintf("gists/%v/commits", id)
+	req, err := s.client.NewRequest("GET", u, nil)
+
+	if err != nil {
```

Comment:
It's better style to handle the error value immediately after `err` is assigned, and not have a blank line in between.

Blank lines in Go are used sparingly, and they are great for separating completely separate blocks of code. This is exactly the opposite of what you want when it comes to checking the value of `err`, since at first glance this may read as you didn't check value of `err` at all.

You should get rid of this blank line.

Look at some of the other methods in this file as an example for spacing. It's better to stay consistent.